.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '[0-9].[0-9]+'
+    tags: [ 'v*' ]
+  pull_request:
+    branches:
+      - master
+      - '[0-9].[0-9]+'
+
+jobs:
+  test:
+    name: Test on Python ${{ matrix.python }}
+    strategy:
+      matrix:
+        python:
+          - 3.6
+          - 3.7
+      fail-fast: false
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Install dependencies
+      run: |
+        pip install -e .
+        pip install -Ur requirements.txt
```

Comment:
Not sure if we should use `-U` here, it can lead to unpredictable errors.
Usually, I follow the schema:
1. `requirements.txt` has strict versions only (`dep-name==1.2.0`).
2. `pip install -r requirements.txt` installs only specified versions
3. `dependabot` creates PR for each dependency update, you can verify if the CI passes the version bump correctly and merge the PR.

Reply:
Thanks for the comment, I've fixed it.